### DISCUSSION TITLE
Strange issue with inputText/inputNumber intermittently not accepting input

### DISCUSSION THREAD

**Discussion by MP2E** [primefaces/community#3154] [PrimeFaces] [answered]
Hi! I'm having an odd issue. Sometimes(like 1% of the time) while clicking a text field on my web app, all text fields break and won't allow me to click them or type into them. Also strange is that, in this broken state, if I click a commandbutton with an action to navigate to a different page on the web app, the input fields are broken on that new page too! The only workaround I've found is to close the web browser tab and reopen it, then navigate back to the web page.

Many of the input text fields(but not all) also have Ajax tags attached to them like the following in case that is relevant:

<p:ajax async="true" event="click" listener="#{fieldListViewBean.handleDisplayFieldClick(widget.name)}" update="@all"/>
<p:ajax async="true" event="change" listener="#{fieldListViewBean.recalculate()}"  update="@all"/>

This issue happens on Safari and Google Chrome, and I used Inspect with the developer console to see what was going on, but the fields don't seem to have the readonly or disabled attributes, so I'm confused as to what could even be causing it.

I'm using Primefaces 14.0.8, but I also experience this issue with Primefaces 13. Probably older versions are affected, too.

Unfortunately I can't share the source of my web app to help diagnose the issue.

Any pointers would be appreciated, thanks for reading.

**Comment by jrgamber** [comment]
A variety of primefaces widgets boil down to have html input tag as part of their html.  
When this issue of locking up occurs, command buttons work ok, but many pf input widgets like spinners not take value changes.
These html input fields will not show a text input cursor when clicked on, and will not take text input.

**Comment by tandraschko** [comment, marked as answer]
try without update=`@all`, that is a anti-pattern and things can break. async=true and update=all sounds like a very risky combination.

**Comment by MP2E** [comment]
Just wanted to say thanks! The problem seems to have gone away with the changes you suggested.
